pip/qsharp/interop/qiskit/backends/backend_base.py

```diff
@@ -319,19 +320,26 @@ def _transpile(self, circuit: QuantumCircuit, **options) -> QuantumCircuit:
 
         circuit = self.run_qiskit_passes(circuit, options)
 
-        orig = self.target.num_qubits
-        try:
```

Comment:
Are you confident we don't need the `try` block anymore?

Reply:
In this case, yes, because all it was doing was making sure the `finally` set the `num_qubits` on the target back to the original value. Now we are instead respecting the target configured value throughout which means a user's custom target either needs to explicitly return `None` (as we now do) or pick a specific value that will cause transpilation to reject a circuit that is too big (which seems to be the newer, desired behavior for Qiskit targets).